internal/cli/rules.go

```diff
@@ -307,10 +307,10 @@ func updateRuleCmd(cli *cli) *cobra.Command {
 			"To update interactively, use `auth0 rules update` with no arguments.\n\n" +
 			"To update non-interactively, supply the rule id and other information through the flags.",
 		Example: `  auth0 rules update <id>
-  auth0 rules update <rule-id> --enabled true
-  auth0 rules update <rule-id> --enabled true --name "My Updated Rule"
-  auth0 rules update <rule-id> --enabled true --name "My Updated Rule" --script "$(cat path/to/script.js)"
-  auth0 rules update <rule-id> -e true -n "My Updated Rule" -s "$(cat path/to/script.js)" --json
+  auth0 rules update <rule-id> --enabled=true
```

Comment:
Maybe let's not use examples that have `=true` (the default when the flag is passed without an explicit bool value), and only use `=` for `=false` examples?
Like in the examples for `auth0 users import` in the file below.

Reply:
Same for all the other instances.

Reply:
Negating the  `--enabled` flag is ambiguous, could be `--disabled` could be `--enabled false` or could be `--enabled=false`. Only one of which is correct. Why not make the template explicitly clear in the example to eliminate any confusion? Can you explain the upside you see by not doing that?

Reply:
Using bool flags like the following:
- Case "true": `--enabled`
- Case "false": `--enabled=false`
Follows the existing conventions for bool flags in other CLIs, as far as I can tell. But of course, feel free to not do it if you disagree.

Reply:
Just to be clear, what I'm suggesting here is to use this form: `--enabled` for "true" examples, and this other form: `--enabled=false` for "false" examples, like in the docs for `auth0 users import`.